ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/RunningQuery.java

```diff
@@ -31,15 +30,15 @@
   private final Set<String> sinks;
   private final Set<String> sinkKafkaTopics;
   private final QueryId id;
-  private final Optional<String> state;
+  private final KafkaStreamsStateCount state;
 
   @JsonCreator
   public RunningQuery(
       @JsonProperty("queryString") final String queryString,
       @JsonProperty("sinks") final Set<String> sinks,
       @JsonProperty("sinkKafkaTopics") final Set<String> sinkKafkaTopics,
       @JsonProperty("id") final QueryId id,
-      @JsonProperty("state") final Optional<String> state
+      @JsonProperty("state") final KafkaStreamsStateCount state
```

Comment:
Might be worth keeping the old state for a while as this will allow backwards compatibility for older CLI versions... i.e. 

```suggestion
      @JsonProperty("state") final Optional<String> state, // Kept for backwards compatibility.
      @JsonProperty("stateCounts") final KafkaStreamsStateCount stateCounts
```

and just populate the `state` string somehow, e.g. using the `stateCounts` string representation?  I *think* this would work as I think the old CLI just outputs this string as-is. Need to check though.

Reply:
yea, I'm refactoring the output so that it'll be more backwards compatible here.